migrations/versions/e1a50dae1ac9_.py

```diff
@@ -18,50 +19,55 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
+    # Users table
     op.create_table('users',
-    sa.Column('created_at', sa.DateTime(), nullable=False),
-    sa.Column('updated_at', sa.DateTime(), nullable=False),
-    sa.Column('deleted_at', sa.DateTime(), nullable=True),
-    sa.Column('id', sa.Integer(), nullable=False),
-    sa.Column('name', sa.String(length=128), nullable=False),
-    sa.Column('email', sa.String(length=128), nullable=False),
-    sa.Column('password', sa.String(length=128), nullable=True),
-    sa.Column('admin', sa.Boolean(), server_default=sa.text('false'), nullable=False),
-    sa.Column('super_admin', sa.Boolean(), server_default=sa.text('false'), nullable=False),
-    sa.PrimaryKeyConstraint('id'),
-    sa.UniqueConstraint('email')
-    )
+                    sa.Column('created_at', sa.DateTime(), nullable=False),
```

Comment:
I know this is automatic but it's weird that they have that much indentation...

Reply:
It the indentation changed by pycharm in the file. It is its standard formatting